catalog/core/catalog-core-api/src/main/java/ddf/catalog/content/operation/CreateStorageRequest.java

```diff
@@ -25,7 +24,7 @@
  * removed in a future version of the library. </b>
  * </p>
  */
-public interface CreateStorageRequest extends Request {
+public interface CreateStorageRequest extends StorageRequest {
 
     /**
      * Returns {@link List} of {@link ContentItem}
```

Comment:
Instead of repeating the same thing here, should we just keep the `@return`? We should also specify if the list returned by `getContentItems()` can be `null`.